src/main/build/setup-jython-env.sh

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
+set -e;
```

Comment:
This works fine, but you can do this on the shebang line:
```shell
#!/bin/bash -e
```
(I like to use `-eu` for most shell scripts, as `-u` prevents typos for variable names.)